app/main.py

```diff
@@ -1,8 +1,79 @@
 class Car:
-    # write your code here
-    pass
+    """
+    comfort_class - comfort class of a car, from 1 to 7
+    clean_mark - car cleanness mark,
+    from very dirty - 1 to absolutely clean - 10
+    brand - brand of the car
+    """
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
```

Comment:
Let's stick to the same code style.
If you want to have another argument on a line with (, then the last “)” should be next to the argument.
There are options in this situation:

def __init__(self,
                 comfort_class: int,
                 clean_mark: int,
                 brand: str) -> None:
                 
                or (my recommendation)
def __init__(
    self,
    comfort_class: int,
    clean_mark: int,
    brand: str
) -> None:

Reply:
Also, don't forget to fix it in other places